Database/ReleaseScript/0617 -Add contact details to region.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE dbo.DNRUplandRegion
+ADD RegionAddress varchar(255) NULL,
+	RegionCity varchar(30) NULL,
+	RegionState varchar(30) NULL,
+	RegionZip varchar(10) NULL,
+	Phone varchar(30) NULL,

```

Comment:
For consistency, I think we should make this RegionPhone and RegionEmail. This also makes them clearly different from the Phone/Email fields on the Person table